arch/arm/src/stm32h7/stm32_sdmmc.c

```diff
@@ -381,10 +381,14 @@ struct stm32_dev_s
 #if !defined(CONFIG_STM32H7_SDMMC_IDMA)
   struct work_s      cbfifo;          /* Monitor for Lame FIFO */
 #endif
-  uint8_t            rxfifo[FIFO_SIZE_IN_BYTES] /* To offload with IDMA */
-                     aligned_data(ARMV7M_DCACHE_LINESIZE);
-#if defined(CONFIG_ARMV7M_DCACHE) && defined(CONFIG_STM32H7_SDMMC_IDMA)
-  bool               unaligned_rx; /* read buffer is not cache-line aligned */
+  uint8_t            rxfifo[FIFO_SIZE_IN_BYTES] /* To offload with IDMA and support un-alinged buffers */
+                     __attribute__((aligned(ARMV7M_DCACHE_LINESIZE)));
```

Comment:
Maybe use a macro from compiler.h?
Or alloc and uint32_t arrays for rounded up size in bytes divided by 4.

Reply:
Done